### PR TITLE
removed unwanted response fields from qr code response

### PR DESCRIPTION
**Pull request by javidlulu**
### All Submissions:

* [ ] Have you followed the guidelines in our [Contributing](/contributing.md) document?
* [ ] Have you read and signed the automated Contributor's License Agreement?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### New Feature Submissions:

1. [ ] Does your submission pass tests?
2. [ ] Have you lint your code locally prior to submission?

### Changes to Core Features:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: src/main/java/jp/ne/paypay/model/QRCodeResponse.java [hunk 1]
```diff
@@ -120,12 +120,6 @@ public RedirectTypeEnum read(final JsonReader jsonReader) throws IOException {
   @SerializedName("authorizationExpiry")
   private Integer authorizationExpiry = null;
   
-  @SerializedName("amountescription")
```
Comment (shreyanshp) [inline src/main/java/jp/ne/paypay/model/QRCodeResponse.java:123]: @javidlulu check this once in OPA API doc and confirm that this field is removed both from request and response?

Comment (javidlulu) [reply src/main/java/jp/ne/paypay/model/QRCodeResponse.java:123]: its not there in request and response https://www.paypay.ne.jp/opa/doc/v1.0/dynamicqrcode#operation/createQRCode.
Removed from SDK here.

**Comment by javidlulu** [comment]
codeclimate issue on duplicates is on models, but are different model objects

File: src/main/java/jp/ne/paypay/model/PaymentOrderDetails.java [hunk 2]
```diff
@@ -1,273 +0,0 @@
...
-import java.util.List;
-import java.util.Objects;
-
-import io.swagger.annotations.ApiModelProperty;
-
-/**
- * PaymentOrderDetails
- */
-
-public class PaymentOrderDetails {
```
Comment (shreyanshp) [inline src/main/java/jp/ne/paypay/model/PaymentOrderDetails.java:15]: This is no longer used?

Comment (javidlulu) [reply src/main/java/jp/ne/paypay/model/PaymentOrderDetails.java:15]: No, I have removed other models also which are not used and tested the apis

**Review by shreyanshp — APPROVED** [review]

**Pull request merged**